src/main/java/zmaster587/libVulpes/tile/multiblock/TileMultiBlock.java

```diff
@@ -366,10 +367,10 @@ else if(((IMultiblock)tile).getMasterBlock() == this)
 							continue;
 					}
 					//Make sure the structure is valid
-					if(!(structure[y][z][x] instanceof Character && (Character)structure[y][z][x] == 'c') && !(structure[y][z][x] instanceof Block && (Block)structure[y][z][x] == Blocks.AIR && world.isAirBlock(globalPos)) && !getAllowableBlocks(structure[y][z][x]).contains(new BlockMeta(block,meta))) {
+					if(!(structure[y][z][x] instanceof Character && (Character)structure[y][z][x] == 'c') && !(structure[y][z][x] instanceof Block && structure[y][z][x] == Blocks.AIR && world.isAirBlock(globalPos)) && !getAllowableBlocks(structure[y][z][x]).contains(new BlockMeta(block,meta))) {
```

Comment:
Should probably put this on multiple lines for readability
```suggestion
					if(!(structure[y][z][x] instanceof Character
					&& (Character)structure[y][z][x] == 'c')
					&& !(structure[y][z][x] instanceof Block
					&& structure[y][z][x] == Blocks.AIR
					&& world.isAirBlock(globalPos))
					&& !getAllowableBlocks(structure[y][z][x]).contains(new BlockMeta(block,meta))) {
```